quilt/tools/build.py

```diff
@@ -20,8 +20,9 @@
 from . import check_functions as qc            # pylint:disable=W0611
 
 def _is_internal_node(node):
-    # all of an internal nodes children are dicts
-    return all(isinstance(x, dict) for x in node.values())
+    # at least one of an internal nodes children are dicts
+    # some (group args) may not be dicts
+    return any(isinstance(x, dict) for x in node.values())
```

Comment:
I forgot how this logic works, but do we care about empty lists? (They will be rejected by `any`.)

Reply:
Also, might as well use `itervalues(node)`.

Reply:
I tested this and `any` doesn't care if the dict child is empty. It will return True. Not sure I understand the concern. BTW if there is an empty dict the leaf node code will throw an exception because it doesn't have a `file` key.

Reply:
I meant, if `node.values()` returns an empty list: `all([])` is `True`, but `any([])` is `False`. Anyways, if it works, that's fine